doc/nrf/protocols/matter/end_product/versioning.rst

```diff
@@ -0,0 +1,87 @@
+.. _ug_versioning_in_matter:
+
+Maintaining firmware version
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Device Firmware Update (DFU) is the procedure of upgrading the application firmware version on a device.
+For example, you can transfer the updated images to the device by:
+
+* Upgrade over Bluetooth® Low Energy (LE).
+* :ref:`Upgrade over Matter (Matter OTA)<ug_matter_overview_dfu>`, where the updates are sent over the Matter fabric.
+
+Before creating and performing DFU, you need to prepare a proper versioning configuration.
+Versioning can be maintained by creating a :ref:`VERSION<app-version-details>` file, or by using Kconfig options.
```

Comment:
I wonder if we need description of DFU. At this point user is familiar with DFU procedure from the previous documentation pages.

What about to change this entire paragraph to something like this:
```suggestion
To effectively deploy a Matter product, it is essential to implement application version management. This versioning is crucial for managing firmware upgrades on devices, and it is also displayed within ecosystem applications, as provided by the Basic Information cluster.

There are two primary approaches for maintaining versioning. It's important to note that these approaches should not be used simultaneously:

Utilizing a VERSION file, as detailed in the Zephyr Project documentation: [Zephyr Project Versioning](https://docs.zephyrproject.org/latest/build/version/index.html#). This method involves defining the version information in a specific file format.

Implementing dedicated Kconfig configurations. This approach uses Kconfig system configurations to set and manage the versioning details.

Choose the approach that best aligns with your project requirements and infrastructure.
```